_data/akulkarnihawk1995/akulkarnihawk1995.md

```diff
@@ -0,0 +1,6 @@
+---
+name: Akshay Kulkarni # No longer than 28 characters
+institution: Illinois Tech 🚩 # no longer than 58 characters
+quote: Whenever I’m about to do something, I think, Would an idiot do that? - Dwight Schrute # no longer than 100 characters, avoid using quotes(") to guarantee the format remains the same.
```

Comment:
We will need you to change the quote on the file to fit in our Code of Conduct.

Reply:
How do I change the quote of already merged pull request ?